test/system-test/test_unwrapkey.py

```diff
@@ -0,0 +1,105 @@
+import json
+import pytest
+from endpoints import key, refresh, unwrapKey
+from utils import apply_kms_constitution, apply_key_release_policy, trust_jwt_issuer, get_test_attestation, get_test_public_wrapping_key, decrypted_wrapped_key, apply_settings_policy
+
+
+def test_unwrap_key_and_decrypt(setup_kms):
+    apply_kms_constitution()
+    apply_key_release_policy()
+    apply_settings_policy()
+    refresh()
+    while True:
+        status_code, key_json = key(
+            attestation=get_test_attestation(),
+            wrapping_key=get_test_public_wrapping_key(),
+        )
+        if status_code != 202:
+            break
+    assert status_code == 200
+
+    # unwrap key
+    status_code, unwrapped_json = unwrapKey(
+        attestation=get_test_attestation(),
+        wrapping_key=get_test_public_wrapping_key(),
+        wrappedKid=key_json["wrappedKid"]
+    )
+    assert status_code == 200
+    unwrapped = decrypted_wrapped_key(unwrapped_json["wrapped"])
```

Comment:
It took me a second to remind myself of the unusual scheme of `/key` just returns a KID and then `/unwrapKey` returns the actual key (which by the looks of it is still wrapped?) 

Can you add a comment explaining what this test is expecting from the endpoint and how we're validating that it works correctly?

Reply:
Done in https://github.com/microsoft/azure-privacy-sandbox-kms/pull/288/commits/2809bbb0936ca82c4fa74cca193e3095bf7ea6dd